data/js/main.js

```diff
@@ -1,7 +1,237 @@
-﻿$(document).ready(function(){
-    $('.collapse').collapse()
-    $('#user-rating-form').on('change','[name="rating"]',function(){
-        $('#selected-rating').text($('[name="rating"]:checked').val());
-    });
+﻿'use strict';
+// $(document).ready(function () {
+//
+//     $('.collapse').collapse()
+//     $('#user-rating-form').on('change', '[name="rating"]', function () {
+//         $('#selected-rating').text($('[name="rating"]:checked').val());
+//     });
+//
+// });
+
+let catalog = [
+    {
+        title: 'MANGO PEOPLE T-SHIRT',
+        color: 'red',
+        size: 'XS',
+        price: 5,
+        url: './data/img/product_1.jpg',
+        alt: 'MANGO PEOPLE T-SHIRT',
+        quantity: 1
+    },
+    {
+        title: 'MANGO PEOPLE DRESS',
+        color: 'BLACK',
+        size: 'M',
+        price: 10,
+        url: './data/img/product_2.jpg',
+        alt: 'MANGO PEOPLE DRESS',
+        quantity: 1
+    },
+    {
+        title: 'MANGO PEOPLE SKIRT',
+        color: 'WHITE',
+        size: 'S',
+        price: 15,
+        url: './data/img/product_3.jpg',
+        alt: 'MANGO PEOPLE SKIRT',
+        quantity: 1
+    }
+];
+let goodsInDrop = document.querySelector('.drop-wrap-cart');
```

Comment:
```suggestion
const goodsInDrop = document.querySelector('.drop-wrap-cart');
```
для всех dom-элементов больше подойдет const